modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -81,6 +81,7 @@ When scaffolding an AsyncAPI specification into a Mule project, Studio:
 * Considers one spec at a time
 * Treats the imported spec as a modification of an existing spec and evaluates it for rescaffolding
 * Creates a new Mule project with a separate flow for each `publish` operation in the specification
+* Generates no flow.xml if the AsyncAPI spec doesn’t include a `publish` operation
```

Comment:
The xml file is generated, but it's empty. Also, just to clarify, the xml that contains the scaffolded flows has the name of the Mule project.

Reply:
OK! Thanks!